docs/source/recipes/plot_13_recipe.py

```diff
@@ -0,0 +1,265 @@
+"""
+Calculate and plot the Niño 3.4 Index
+====================================
+
+In this recipe, we will calculate and plot the sea surface temperature (SST)
+anomaly in the Niño 3.4 region. According to `NCAR Climate Data Guide
+<https://climatedataguide.ucar.edu/climate-data/nino-sst-indices-nino-12-3-34-4-oni-and-tni>`_,
+the Niño 3.4 anomalies may be thought of as representing the average equatorial
+SSTs across the Pacific from about the dateline to the South American coast.
+The Niño 3.4 index typically uses a 5-month running mean, and El Niño or La
+Niña events are defined when the Niño 3.4 SSTs exceed +/- 0.4 degrees C for a
+period of six months or more.
+
+"""
+
+# %%
+# 1. Import cf-python, cf-plot and matplotlib.pyplot:
+
+import cfplot as cfp
+
+import cf
+
+# %%
+# 2. Read and select the SST by index and look at its contents:
+sst = cf.read("~/recipes/ERA5_monthly_averaged_SST.nc")[0]
+print(sst)
+
+# %%
+# 3. Set the units from Kelvin to degrees Celsius:
+sst.Units = cf.Units("degreesC")
+
+# %%
+# 4. SST is subspaced for for Niño 3.4 region (5N-5S, 170W-120W) and as the
+# dataset is using longitudes in 0-360 degrees East format, they are substracted
+# from 360 to convert them:
+region = sst.subspace(X=cf.wi(360 - 170, 360 - 120), Y=cf.wi(-5, 5))
+
+# %%
+# 5. Plot the various Niño regions using cf-plot. Here:
+#
+# - `cfplot.gopen <http://ajheaps.github.io/cf-plot/gopen.html>`_ is used to
+#   define the parts of the plot area, which is closed by
+#   `cfplot.gclose <http://ajheaps.github.io/cf-plot/gclose.html>`_;
+# - `cfplot.mapset <https://ajheaps.github.io/cf-plot/mapset.html>`_ is used to
+#   set the map limits and projection;
+# - `cfplot.setvars <http://ajheaps.github.io/cf-plot/setvars.html>`_ is used to
+#   set various attributes of the plot, like enabling a grid and and setting the
+#   land color to grey;
```

Comment:
```suggestion
#   land colour to grey;
```
to be consistent with later usage (and we generally prefer UK English over American English for spellings with alternatives in the docs, etc.)